### DISCUSSION TITLE
Format align text message in telegram bot

### DISCUSSION THREAD

**Discussion by Truong1803** [telegraf/telegraf#1723] [Q&A] [answered]
How can i send message to telegram bot with below format text ? I can't find solution with option "parse_mode: HTML".
![image](https://user-images.githubusercontent.com/50040527/199377136-9b8207ce-09f1-44f2-9abd-bbf8343bf97c.png)
I am formatting with tabs and spaces but for example the description field , if the field has more than two lines, I cannot format the second line and the next line of the description starts from the first line.

**Comment by MKRhere** [comment, marked as answer]
This is impossible to do because the message bubble can be different widths on different screens, and Telegram and the Bot API do not support tables. If a message is too long, it will wrap.

**Reply by Truong1803** [reply]
Alright, thank you for your reply.
